src/Services/ListenTogether/ListenTogether.Application/Grains/RoomGrain.cs

```diff
@@ -0,0 +1,57 @@
+﻿using ListenTogether.Application.Interfaces;
+using ListenTogether.Domain;
+using Microsoft.Extensions.Logging;
+using Orleans;
+using Orleans.Runtime;
+
+namespace ListenTogether.Application.Grains
+{
+    public class RoomGrain : Grain, IRoomGrain
+    {
+        IPersistentState<Room> Room { get; set; }
+        public ILogger<RoomGrain> Logger { get; }
```

Comment:
```suggestion
        private readonly IPersistentState<Room> _room;
        private readonly ILogger<RoomGrain> _logger;
```

These are better being `readonly` fields, unless there is a good reason to expose them.